### ISSUE TITLE
Add a11y e2e tests

### ISSUE THREAD

**Issue by denyo** [porsche-design-system/porsche-design-system#1564]
Scope
---
During the refactoring of our `p-select-wrapper` we noticed some accessibility issues that were freshly introduced but also existed before. 
Therefore we should capture the state of a11y on our components by a test to avoid regressions.

It appears that puppeteer is offering the tools to accomplish this.

Out of Scope
---
- This e2e test should not replace the additional (future) implementation of AXE Core / Lighthouse (see #461)

Notes
---
- https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#class-accessibility

Acceptance criteria
---
- [x] Components should have tests which covers the rendering of the accessibility tree

Subtasks
---
- [x] Add e2e accessibility tree snapshots to all components
- [x] Check run time for VRT Tests (https://github.com/smooth-code/jest-puppeteer/pull/420)
- [x] Check dependencies of all packages (jest should only be referenced by shared package)
- [x] Check/Equalize TS Configs
- [x] Update OSO Config
- [x] ~Remove 2nd empty browser tab that is opened by default if possible~
- [x] Check DevDependency of Jest 
- [x] #1611
